app/views/projects/_project.html.erb

```diff
@@ -0,0 +1,101 @@
+<div class='project-wrapper'>
+
+  <% if @project.featured_artist_image.present? %>
+    <div class="featured-artist">
+      <div class="row">
+        <div class="example col-md-9 vcenter" id="example">
+          <div class="student-image">
+            <%= image_tag @project.featured_artist_image.url(:medium) %>
+          </div>
+          <div class="student-example">
+            STUDENT EXAMPLE
+          </div>
+          <div class="superhero-quote">
+            <%= @project.featured_artist_description %>
+            -<%= @project.featured_artist_name %>
+          </div>
+        </div>
+      </div>
+    </div>
+  <% end %>
+
+  <hr />
+
+  <h1>Project Overview</h1>
+  <div>
+    <h2 id='project-synopsis'>synopsis</h2>
+    <p><%= @project.synopsis %></p>
+  </div>
+  <hr />
+
+  <h1>Setup At A Glance</h1>
+  <div>
+    <h2 id='project-materials'>materials</h2>
+    <p><%= @project.materials %></p>
+    <div class="project-indent">
+      <small>*suggestion: <%= @project.materials_suggestion %></small>
+    </div>
+
+    <h2 id='project-books-media'>books/media</h2>
+    <p><%= @project.books_media %></p>
+
+    <h2 id='project-photocopies'>photocopies</h2>
+    <div>
+      <% if @project.first_photocopy.present? %>
+        <p><%= @project.first_photocopy_header %></p>
+        <p class="pdf-indent"><%= link_to @project.first_photocopy_file_name, @project.first_photocopy.url %></p>
+      <% else %>
+        <%= "N/A" %>
+      <% end %>
+      <% if @project.second_photocopy.present? %>
+        <p><%= @project.second_photocopy_header %></p>
+        <p class="pdf-indent"><%= link_to @project.second_photocopy_file_name, @project.second_photocopy.url %></p>
+      <% end %>
+      <% if @project.third_photocopy.present? %>
+        <p><%= @project.third_photocopy_header %></p>
+        <p class="pdf-indent"><%= link_to @project.third_photocopy_file_name, @project.third_photocopy.url %></p>
+      <% end %>
+      <p><small><%= @project.photocopy_copyright %></small></p>
+      <small>© info: Doodles Academy presents both public domain artworks and works that are protected by copyright in their videos and lessons. The latter are used in accordance with fair use principle, as the images are only being used for educational purposes. If you are the copyright holder of the work of art used for Doodles-Academy.org and do not agree that the use of your image is a fair use, please contact us by email.</small>
+    </div>
+  </div>
+  <hr />
+
+  <h1>Standards</h1>
+  <div>
+
+    <% if @project.common_core != blank? %>
```

Comment:
@jasonoverby I think you want `if @project.common_core.present?` here, or `unless @project.common_core.blank?`.